sdk/src/connections/usb/linux/device_enumerator_linux.cpp

```diff
@@ -29,19 +29,53 @@
  * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+#include "connections/usb/usb_utils.h"
 #include "device_enumerator_impl.h"
-#include "utils_linux.h"
+#include "usb_linux_utils.h"
+#include "utils.h"
 
 #include <dirent.h>
 #include <fcntl.h>
 #include <glog/logging.h>
 #include <linux/videodev2.h>
 #include <sys/stat.h>
 
-#define CLEAR(x) memset(&(x), 0, sizeof(x))
+using namespace std;
+
+static aditof::Status getAvailableSensors(int fd,
+                                          string &advertisedSensorData) {
+    int ret;
```

Comment:
Separate declaration and assignment, maybe `int ret = ...`

Reply:
You mean to declare and assign in a single line (not to separate). Fixed.